AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/prefab_utils.py

```diff
@@ -156,6 +156,19 @@ def get_direct_child_entity_by_name(self, name: str):
             if entity.get_name() == name:
                 return entity
 
+    def get_child_entity_by_name(self, name: str) -> EditorEntity:
+        """
+        Returns the child entity in the current prefab instance with the given name. Will search from the container
+        entity root through all children.
+        :param name: The name of the child entity to find
+        :return: EditorEntity
+        """
+        search_filter = azlmbr.entity.SearchFilter()
+        search_filter.names = [name]
+        search_filter.roots = [self.container_entity.id]
+        search_filter.names_are_root_based = False
+        return EditorEntity(azlmbr.entity.SearchBus(bus.Broadcast, 'SearchEntities', search_filter)[0])
```

Comment:
This assumes a child entity with this name exists. Might be nice to add a quick check that the list returned from `SearchEntities` isn't empty, and if it is empty than return `None`. Or could add a comment in the description such as: "NOTE: This method assumes the container has a child with matching name"

Reply:
Added a check that can return None.